molgenis-python/src/main/resources/templates/molgenis.py.ftl

```diff
@@ -158,3 +175,18 @@ class Session():
         z = x.copy()
         z.update(y)
         return z
+
+    def _raise_for_status(self, response):
+        error = None
+        try:
+            if 'errors' in response.json() and 'message' in response.json()['errors'][0]:
+                error = (response.json()['errors'][0]['message'])
+        except json.decoder.JSONDecodeError:
+            pass
+        try:
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as e:
+            if error:
+                raise requests.exceptions.HTTPError(str(e)+'\n'+error)
```

Comment:
The message and details are in str(e). We could also use import traceback traceback.print_exc(), or some other solution. Because with response.raise_for_status() you will only get "<Response 404> Bad request", with no information about why it is a bad request.